torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -1136,6 +1357,7 @@ void LLVMCodeGenImpl::visit(const For* v) {
   auto exit = llvm::BasicBlock::Create(getContext(), "exit", fn_);
 
   // Create the stop condition.
+  irb_.SetInsertPoint(condBlock);
```

Comment:
Is this needed?  I'm a little surprised since we just do `irb_.SetInsertPoint(condBlock);` a few lines ago (1344) and I don't see anything that would invalidate that...